source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -260,11 +260,11 @@ After adding a repository to the `openedx-translations repo`_ verify the followi
 .. _chore - add updated translation source files #615: https://github.com/openedx/openedx-translations/pull/615
 .. _Updates for file translations/frontend-app-learning/src/i18n/transifex_input.json in de on branch main #598: https://github.com/openedx/openedx-translations/pull/598
 .. _course discovery translations: https://github.com/openedx/openedx-translations/tree/f0315d4/translations/course-discovery/course_discovery/conf/locale
-.. _frontend-lib-special-exams testing pull request: https://github.com/Zeit-Labs/openedx-translations/pull/1/files
 .. _transifex.yml: https://github.com/openedx/openedx-translations/blob/main/transifex.yml
 .. _Drag and Drop XBlock transifex.yml entry: https://github.com/openedx/openedx-translations/blob/19c0fcbbc334c56022df355fa5b529e5853d30f9/transifex.yml#L253-L259
-.. _edx-platform e7fc0c6: https://github.com/openedx/edx-platform/commit/e7fc0c6b6f8b44fa4bcc71d00ae2931f91cc664c
+.. _edx-platform Redwood: https://github.com/openedx/edx-platform/commit/e7fc0c6b6f8b44fa4bcc71d00ae2931f91cc664c
```

Comment:
Should this be a tag?

Reply:
I think the naming is just confusing here. The link is to the commit `feat: \`atlas pull\` for XBlock translations | FC-0012`. Linking to the tagged version wouldn't provide the same context as linking to this commit.

I think another possible option would be to link to the PR (https://github.com/openedx/edx-platform/pull/33698), so up on line 48 where it says "Redwood or newer" we could change that to be "A version of edx-platform that includes PR 33698 (Redwood or newer)."

Reply:
Fixed, now Redwood is behind us. Removing the link altogether makes more sense.